### DISCUSSION TITLE
Pagination for a table with a custom SQL query

### DISCUSSION THREAD

**Discussion by EvanGllt** [Budibase/budibase#14134] [Help]
Hello, there! 👋

I am currently looking to set up pagination for my table using a custom SQL query. My database is in MS SQL Server.
By consulting the forums, I understood that automatic pagination is not available and that I had to do it myself. I even tried to follow this tutorial (https://github.com/Budibase/budibase/discussions/7313#discussioncomment-3430353) but I got the error `Query Error: Incorrect syntax near 'LIMIT'.` and I don't understand why.

[![aide 1](https://github.com/Budibase/budibase/assets/173894293/3a81e513-8181-4360-a3ad-99e2e503a64e)](url)

Could you help me? Thanks in advance.

**Comment by mikesealey** [comment]
Hey @EvanGllt 

At first glance, it appears that the issue could be that your binding name is `limite` but on line 19 of your query you call `{{ limit }}`

**Reply by EvanGllt** [reply]
Yes, I've seen that in the meantime, but it's not that, it doesn't change anything, I have the same error.
